File: 프로그래머스/lv0/120904. 숫자 찾기/숫자 찾기.py
def solution(num, k):
    return -1 if str(k) not in str(num) else str(num).find(str(k))+1

# find() : 찾는 문자의 인덱스 반환 없으면 -1 반환
# index() : 찾는 문자의 인덱스 반환 없으면 에러
# 인덱스 반환이라 0부터 시작이니까 자릿수는 +1

File: 프로그래머스/lv0/120904. 숫자 찾기/숫자 찾기.java
class Solution {
    public int solution(int num, int k) {
        int answer = -1;
        String sNum[] = String.valueOf(num).split("");
        //num을 String으로 변환 후 split => 배열로 저장
		for(int i=0; i<sNum.length; i++) {
			if(sNum[i].equals(String.valueOf(k))) {
				answer = i+1; //i가 index값이므로 +1 해주기
				break;  
            }
		}
        return answer;
    }
}